src/util.js

```diff
@@ -3,7 +3,7 @@ import varlink from './varlink.js';
 
 const _ = cockpit.gettext;
 
-export const PODMAN_ADDRESS = "unix:/run/podman/io.podman";
+export const PODMAN_ROOT_ADDRESS = "unix:/run/podman/io.podman";
```

Comment:
Bikeshedding: This is a bit confusing, as "root" is also a user. How about `PODMAN_SYSTEM_ADDRESS`? system and user is the dichtonomy that systemd uses; libvirt has "system" and "session", so at least vaguely the same.

Reply:
Changed to `PODMAN_SYSTEM_ADDRESS`